cli/endpoints/amlarc/managed/canary-declarative-flow/1-create-endpoint-with-blue.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/onlineEndpoint.schema.json
+name: my-endpoint
+type: online
+auth_mode: aml_token
+traffic:
+  blue: 100
+identity:
+  type: system_assigned
+target: azureml:/subscriptions/{{subscriptionId}}/resourceGroups/{{resourceGroup}}/providers/Microsoft.MachineLearningServices/workspaces/{{workspaceName}}/computes/{{computeName}}
```

Comment:
I prefer the short format.
azureml:{{computeName}}

Reply:
changed to {{fullyQualifiedComputeId}} for clarity for our customers